website/docs/r/api_management.html.markdown

```diff
@@ -0,0 +1,223 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_api_management"
+sidebar_current: "docs-azurerm-resource-api-management-x"
+description: |-
+  Create a Api Management service.
+---
+
+# azurerm_api_management
+
+Create a Api Management component.
+
+## Example Usage (Developer)
+
+```hcl
+resource "azurerm_resource_group" "test" {
+  name     = "api-rg-dev"
+  location = "West Europe"
+}
+
+resource "azurerm_api_management" "test" {
+  name                = "api-mngmnt-dev"
+  publisher_name      = "My Company"
+  publisher_email     = "company@terraform.io"
+  sku {
+    name = "Developer"
+  }
+  location            = "${azurerm_resource_group.test.location}"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+}
+```
+
+## Example Usage (Complete)
+
+```hcl
+resource "azurerm_resource_group" "west" {
+  name     = "api-rg-premium-west"
+  location = "West Europe"
+}
+
+resource "azurerm_resource_group" "north" {
+  name     = "api-rg-premium-north"
+  location = "North Europe"
+}
+
+resource "azurerm_api_management" "test" {
+  name                          = "api-mngmnt"
+  publisher_name                = "My Company"
+  publisher_email               = "company1@terraform.io"
+  notification_sender_email     = "api@terraform.io"
+
+  additional_location {
```

Comment:
could we take this example and run it through terraform fmt to get the correct indentation?